instant/flags.vim

```diff
@@ -51,3 +51,11 @@ call s:plugin.Flag('clang_format_style', 'file')
 " "goimports" (https://godoc.org/golang.org/x/tools/cmd/goimports) to
 " additionally adjust imports when formatting.
 call s:plugin.Flag('gofmt_executable', 'gofmt')
+
+""
+" The path to the js-beautify executable.
+call s:plugin.Flag('js_beautify_executable', 'js-beautify')
+
+""
+" Default formatter for javascript files, 'clang_format' or 'js_beautify'.
+call s:plugin.Flag('javascript_formatter', 'clang_format')
```

Comment:
about lists, yesterday before the sleep i was thinking about it, but in my mind it was more like whitelist, not blacklist, ie to have it described like `l:preferred_formatters_js= [ 'js_beautify', 'clang_format' ]`(btw is it possible to store list as a flag?) and logic of codefmt#IsFormatterAvailable() to select most preferred formatter of all available ones